src/styles/main/_year-control.scss

```diff
@@ -65,3 +65,63 @@
     }
   }
 }
+
+.year-dropdown {
+  display: none;
+}
+
+@media only screen and (max-width: 850px) {
+
+  .year-control {
+    display: none;
+  }
+  
+  #year-select-container {
+    width: 100px;
+    height: 100%;
+  }
+
+  .year-dropdown {
+    display: block;
+  }
+
+  .current-year {
+    position: absolute;
+    font-size: 20px;
+    top: 17%;
+    left: 40%;
+  }
+
+  .far {
+    position: absolute;
+    top: 20%;
+    left: 10%;
+    font-size: 25px;
+  }
+  .year-dropdown-control {
+    display: none;
+    position: absolute;
+    border: 1px solid black;
+    border-top: none;
+    color: #0047ba;
```

Comment:
This should be defined in terms of the theme colour, not a direct blue. See `bitters-seattle/_variables.scss`. There ought to be one in there for the background colour too.